static/app/views/starfish/views/screens/screenLoadSpans/samples/samplesContainer.tsx

```diff
@@ -66,28 +69,51 @@ export function ScreenLoadSampleContainer({
   const {data: spanMetrics} = useSpanMetrics(
     groupId,
     filters,
-    [`avg(${SPAN_SELF_TIME})`, SPAN_OP],
+    [`avg(${SPAN_SELF_TIME})`, 'count()', SPAN_OP],
     'api.starfish.span-summary-panel-samples-table-avg'
   );
 
   return (
     <Fragment>
       <PaddedTitle>
-        {sectionTitle && <SectionTitle>{sectionTitle}</SectionTitle>}
         {release && (
-          <Version organization={organization} version={release} tooltipRawVersion />
+          <SectionTitle>
+            <Tooltip title={release}>
+              <Link
+                to={{
+                  pathname: `/organizations/${organization?.slug}/releases/${encodeURIComponent(
+                    release
+                  )}/`,
+                }}
+              >
+                {release && centerTruncate(release)}
```

Comment:
```suggestion
                {centerTruncate(release)}
```
`release &&` is already checked on line 79